data/hai/hai reveal 5 philosophers.txt

```diff
@@ -807,17 +815,87 @@ ship "Centipede" "Centipede (Teeneep Jump)"
 		"Outfits Expansion"
 		`"Biroo" Atomic Steering`
 		"Hai Diamond Regenerator"
-		"Hai Williwaw Cooling" 2
-		"Hai Chasm Batteries" 2
+		"Hai Corundum Regenerator"
+		"Hai Chasm Batteries"
+		"Liquid Nitrogen Cooler"
 		"Boulder Reactor"
 		Brig
-		"Fuel Pod" 3
+		"Fuel Pod" 2
+		Supercapacitor
 		"A370 Atomic Thruster"
 		"Pulse Rifle" 131
 
 
 
+# Used in the Hai Reveal campaign, Danforth pulls strings with the Free Worlds to get some cruisers equipped for deep-range deployment
+ship "Cruiser" "Cruiser (Quantum Keystone)"
+	outfits
+		"A520 Atomic Thruster"
+		"A525 Atomic Steering"
+		Afterburner
+		"Armageddon Core"
+		"Catalytic Ramscoop"
+		"Fission Reactor"
+		"Fragmentation Grenades" 20
+		"Heavy Anti-Missile Turret" 3
+		"Jump Drive"
+		"LP072a Battery Pack"
+		"LP036a Battery Pack"
+		"Laser Rifle" 20
+		"Liquid Helium Cooler" 2
+		"Outfits Expansion" 2
+		"Plasma Cannon" 6
+		"Quantum Keystone"
+		"S-970 Regenerator"
+		"Fuel Pod" 2
+
+
+ship "Dreadnought" "Dreadnought (Quantum Keystone)"
+	outfits
+		"Catalytic Ramscoop"
+		"D94-YV Shield Generator"
+		"Heavy Anti-Missile Turret" 2
+		"Jump Drive"
+		"LP072a Battery Pack"
+		"Laser Rifle" 25
+		"Liquid Helium Cooler"
+		"Cooling Ducts"
+		"Orca Plasma Steering"
+		"Orca Plasma Thruster"
+		"Outfits Expansion" 4
+		"Fuel Pod" 2
+		"Plasma Turret" 3
+		"Quantum Keystone"
+		"Stack Core"
+		Torpedo 120
+		"Torpedo Launcher" 4
+	turret "Plasma Turret"
+	turret "Heavy Anti-Missile Turret"
+	turret "Heavy Anti-Missile Turret"
+	turret "Plasma Turret"
+	turret "Plasma Turret"
+
+
+ship "Auxiliary" "Auxiliary (Quantum Keystone)"
+	outfits
+		"Sidewinder Missile Launcher" 2
+		"Sidewinder Missile" 90
+		"Quad Blaster Turret" 2
+		"Heavy Anti-Missile Turret" 2
+		"Catalytic Ramscoop"
+		"Armageddon Core"
+		"LP144a Battery Pack"
+		"S-970 Regenerator" 3
+		"Liquid Helium Cooler"
+		"Large Radar Jammer" 2
+		"X4700 Ion Thruster"
+		"X1700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Jump Drive"
+		"Quantum Keystone"
+
```

Comment:
See #7456

Reply:
The variants died quickly, especially the Auxiliary.

Reply:
I'm moving my updated outfits to your variants in variants.txt.